common/src/test/java/org/alfresco/hxi_connector/common/repository/filter/CollectionFilterTest.java

```diff
@@ -67,7 +72,22 @@ void testAspectFiltering(boolean expected, Set<String> aspects, List<String> all
         }
     }
 
-    private static Stream<Arguments> provideParameters()
+    @ParameterizedTest
+    @MethodSource("provideAncestorParameters")
+    void testAncestorFiltering(boolean expected, List<String> ancestors, List<String> allowed, List<String> denied)
+    {
+        boolean result = CollectionFilter.filter(ancestors, allowed, denied);
+        if (expected)
+        {
+            assertTrue(result);
+        }
+        else
+        {
+            assertFalse(result);
+        }
```

Comment:
```suggestion
        assertEquals("Unexpected result from filter", result, expected);
```

Reply:
I thought about that when I was writing this assertion but somehow wanted to stick to `assertTrue`/`assrtFalse` approach with `boolean` assertion. But it does look much neater. I'll change that.